runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkStreamingPortablePipelineTranslator.java

```diff
@@ -433,12 +433,17 @@ private void translateImpulse(
       throw new RuntimeException(e);
     }
 
-    String inputPCollectionId = Iterables.getOnlyElement(transform.getInputsMap().values());
+    String inputPCollectionId = stagePayload.getInput();
+    // TODO: BEAM-2930
```

Comment:
This is easier to reference if you include a full link rather than just the issue number.

Reply:
fixed